### DISCUSSION TITLE
How to add transitions between screens that share TopBar and BottomBar?

### DISCUSSION THREAD

**Discussion by jhernandez06** [adrielcafe/voyager#327] [General]
I am testing Voyager in the navigation of an application that shares the same top bar and bottom bar following the example below and I am not sure how to add transitions between screens correctly, has any of you tried to do it?

```kotlin
@Composable
override fun Content() {
    Navigator(HomeScreen) { navigator ->
        Scaffold(
            topBar = { /* ... */ },
            content = { CurrentScreen() },
            bottomBar = { /* ... */ }
        )
    }
}
```
_This code is in the [Voyager documentation.](https://voyager.adriel.cafe/navigation)_

**Comment by GuizaoBR** [comment]
You find some way to do that? I can only add transitions between Screens and not tabs

**Reply by Velord** [reply]
check this out https://github.com/adrielcafe/voyager/pull/328 look for `TransitionTab `class
